CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to this project will be documented in this file.
 
 - Filter out `CONTRIBUTE_AND_FINALIZE` tasks when detecting missed `REVEALED` status update. (#658)
 - Fetch `results` on-chain when updating a replicate status in `CONTRIBUTE_AND_FINALIZE` workflow. (#659 #660)
+- Fix enclave challenge generation failure. (#663)
```

Comment:
Properly catch all exceptions when an enclave challenge generation fails ?
The idea behind this suggestion is to give an idea of the fix

Reply:
Sure, please see https://github.com/iExecBlockchainComputing/iexec-core/pull/663/commits/a8997d4d9aca42f3c13989da2da320a2f9f2b8a6.